migrations/20170203233255-addSortByToSubject.js

```diff
@@ -10,8 +10,9 @@ module.exports = {
     return qi.sequelize.transaction(() => qi.addColumn('Subjects', 'sortBy', {
       type: Sequelize.STRING(constants.fieldlen.sortField),
       allowNull: true,
+      defaultValue: '',
```

Comment:
you can't modify the migration--it's already been deployed
if you need more db changes, you have to create a new migration